docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+# Copy this into a file called `docker-compose.yml`
+version: "3.2"
+
+    services:
+      todo_app_prod:
+        container_name: production
+        image: todo-app:prod
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
```

Comment:
`type: volume` means that Docker will manage the files, instead of letting you choose a folder. You just specify a name instead. That's not what we want. As the pdf suggests, what you want is a "bind mount" in order to specify a folder you want to share with the container.